### ISSUE TITLE
Generator transforms <a> in <a/> - makes meteor crash

### ISSUE THREAD

**Issue by l1br3** [perak/kitchen-site#309]
Hi,

For an unknown reason the generator transforms this code 

``` javascript
<template name="user_details_moving_link_1">
  <div class="form-group  field-profile-asked-question">
    <label for="profile.askedQuestion">
      View moving
    </label>
    <div class="input-div">
      <p class="form-control-static">
        <a id="movingLink" href="/admin/movings/{{admin_user.movingId}}">View moving</a>
      </p>
    </div>
  </div>
</template>
```

in this code 

``` html
<template name="user_details_moving_link_1">
    <div class="form-group  field-movingId">
        <div class="input-div">
            <p class="form-control-static">
                <a href="/admin/movings/{{admin_user.movingId}}" />             View moving
            </p>
        </div>
    </div>
</template>
```

This is a component to solve the custom_input issue [307](https://github.com/perak/kitchen-site/issues/307)

**Comment by l1br3** [comment]
I'ce successfully hacked the generator by adding a random attribute but it looks like a bug

```html
<template name="user_details_moving_link_1">
  <div class="form-group  field-movingId">
    <div class="input-div">
      <p class="form-control-static">
        <a href="/admin/movings/{{admin_user.movingId}}" attributeAgainstSelfClosingTagBug>
          View moving
        </a>
      </p>
    </div>
  </div>
</template>
```

**Comment by l1br3** [comment]
@perak also listed in my "hacks" section

**Comment by perak** [comment]
@l1br3 This ugly bug is fixed now. Will be available in `0.9.74` release.
